modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/jwt/JWTTestCase.java

```diff
@@ -206,7 +202,7 @@ public void testEnableJWTAndClaims() throws Exception {
      * and is hardcoded to 15 mins. So the test case will take approximately 15mins to complete and it will delay the
      * product build unnecessarily, hence the test case is disabled.
      */
-    @Test(groups = { "wso2.am" }, description = "JWT Token generation when JWT caching is enabled", enabled = false)
+    @Test(groups = { "noRestart" }, description = "JWT Token generation when JWT caching is enabled", enabled = false)
```

Comment:
Why this test is disabled? I can understand the comment saying why we are disabling, but then what is the point of having this test? How do we test the invalidation functionality if we are disabling this test?

Reply:
This is now enabled and fixed it by reducing the cache timeout in api-manager.xml. Fixed with 4f6054ea9733fd040759435c990d6a53cfb56c7a